src/ebonite/core/objects/wrapper.py

```diff
@@ -28,52 +35,126 @@ class ModelWrapper(EboniteParams):
     Must be pyjackson-serializable
     """
     type = None
+    methods_json = 'methods.json'
 
     def __init__(self):
         self.model = None
+        self.methods: typing.Optional[Methods] = None
 
-    @abstractmethod
+    @contextlib.contextmanager
     def dump(self) -> FilesContextManager:
+        with self._dump() as artifact:
+            yield artifact + Blobs({self.methods_json: InMemoryBlob(dumps(self.methods).encode('utf-8'))})
+
+    @abstractmethod
+    def _dump(self) -> FilesContextManager:
         """
         Must return context manager with :class:`~ebonite.core.objects.ArtifactCollection`
         :return: :class:`~ebonite.core.objects.ArtifactCollection`
         """
         pass
 
-    @abstractmethod
     def load(self, path):
+        self._load(path)
+        with open(os.path.join(path, self.methods_json), 'r', encoding='utf-8') as f:
```

Comment:
there is pyjackson.read helper method for that

Reply:
fixed